modules/promotions/promotion-data.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace Elementor\Modules\Promotions;
+
+use Elementor\Core\Utils\Promotions\Filtered_Promotions_Manager;
+use Elementor\Data\EditorAssets\EditorAssetsAPI;
+use Elementor\Utils;
+
+class PromotionData extends EditorAssetsAPI {
```

Comment:
why we need `EditorAssetsAPI`? 

if it's just a client to get data from API/save to transient - it's better to go with composition, not inheritance.

Reply:
I see that this is old code that was moved here, so maybe it's not so critical. But I would love to see here DI usage.

Reply:
@mykytamurzin Implemented